<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string> 串口</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="dataShowForm" name="widget_datashow" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>CAN</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="canDataShowForm" name="widget_canDataShow" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>天太电机展示</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="viewShowForm" name="widget_viewShowForm" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>RMD电机</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="rmdSettingForm" name="widget_rmd" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string> 网络</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="ClientWidget" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string> 零差云控</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="lingChaCanform" name="widget_lingcha" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>鑫拓电机</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="xintuoCanForm" name="widget_xintuo" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>转向电机（研控）</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="yankongDataForm" name="widget_yankong" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>测试联调</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="multipleMotorTestForm" name="widget_test" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>dataShowForm</class>
   <extends>QWidget</extends>
   <header>datashowform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>canDataShowForm</class>
   <extends>QWidget</extends>
   <header>canDataShowForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>viewShowForm</class>
   <extends>QWidget</extends>
   <header location="global">viewShowForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>rmdSettingForm</class>
   <extends>QWidget</extends>
   <header>rmdSettingForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClientWidget</class>
   <extends>QWidget</extends>
   <header>network/clientwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lingChaCanform</class>
   <extends>QWidget</extends>
   <header>lingChaCanform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>xintuoCanForm</class>
   <extends>QWidget</extends>
   <header>xintuoCanForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>multipleMotorTestForm</class>
   <extends>QWidget</extends>
   <header location="global">multipleMotorTestForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>yankongDataForm</class>
   <extends>QWidget</extends>
   <header>yankongDataForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
